packages/paste-website/src/pages/components/topbar/index.mdx

```diff
@@ -0,0 +1,207 @@
+import {Menu, MenuItem, useMenuState, MenuBadge} from '@twilio-paste/menu';
+import {
+  ProductSwitcher,
+  ProductSwitcherButton,
+  ProductSwitcherItem,
+  useProductSwitcherState,
+} from '@twilio-paste/product-switcher';
+import {LogoTwilioIcon} from '@twilio-paste/icons/esm/LogoTwilioIcon';
+import {ProductSegmentIcon} from '@twilio-paste/icons/esm/ProductSegmentIcon';
+import {ProductFlexIcon} from '@twilio-paste/icons/esm/ProductFlexIcon';
+import {ProductEmailAPIIcon} from '@twilio-paste/icons/esm/ProductEmailAPIIcon';
+import packageJson from '@twilio-paste/topbar/package.json';
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+import {
+  defaultTopbarExample,
+  defaultAccountSwitcherExample,
+  defaultProductSwitcherExample,
+} from '../../../component-examples/TopbarExamples.ts';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Topbar');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/topbar"
+  storybookUrl="/?path=/story/components-topbar--"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+## Guidelines
+
+### About Topbar
+
+A Topbar is sticky to the top of the screen and includes application-wide controls including User Dialog (coming soon!), [Product Switcher](#product-switcher), [Account Switcher](#account-switcher), and search [Input](/components/input).
+
+## Examples
+
+### Default Topbar
+
+Use a Topbar at the top of your application to house application-wide controls.
+
+Account-specific controls are aligned to the left (e.g. [Account Switcher](#account-switcher), account type indicator [Badges](/components/badge), and possible upgrade actions).
+
+User-specific controls are aligned to the right (e.g. search [Input](/components/input), support [Button](/components/button), [Product Switcher](#product-switcher), and User Dialog (coming soon!)).
+
+<LivePreview scope={{Topbar}} noInline>
+  {defaultTopbarExample}
+</LivePreview>
+
+## Account Switcher
+
+An Account Switcher is a stylized [Menu Badge](/components/menu#menu-badge) with a list of actions related to a user’s accounts.
+
+Examples of account-related actions include navigating to other recently visited accounts, creating an account, and viewing all of a user’s accounts. Other names for accounts in Twilio products include workspaces and projects.
+
+### Accessibility
+
+Refer to the accessibility section of [Menu](/components/menu).
+
+### Examples
+
+Use an Account Switcher to display actions that a user can trigger related to their accounts.
```

Comment:
```suggestion
An Account Switcher is a stylized [Menu Badge](/components/menu#menu-badge) with a list of actions related to a user’s accounts. [Learn more about Account Switcher.](/components/account-switcher)
```

PR #3255 needs to get merged for the link to work, but we could take all the docs out here and redirect to the Account Switcher page.